saliensHack.user.js

```diff
@@ -139,7 +139,31 @@
 
     var joinZone = function(zoneId) {
         if (joiningZone) return;
+        console.log('--------------------------------');
+        console.log('-------------FIGHT--------------');
+        console.log('--------------------------------');
         console.log('Joining zone:', zoneId);
+        console.log('Uncaptured Zones:', gGame.m_State.m_PlanetData.zones.length);
```

Comment:
I think this is the total zones. You need to filter on the captured flag of the zone object.

```js
gGame.m_State.m_PlanetData.zones.filter(function(zone) { return !zone.captured; }).length
```